consul/resource_consul_keys.go

```diff
@@ -19,6 +19,13 @@ func resourceConsulKeys() *schema.Resource {
 		SchemaVersion: 1,
 		MigrateState:  resourceConsulKeysMigrateState,
 
+		CustomizeDiff: func(d *schema.ResourceDiff, _ interface{}) error {
```

Comment:
Was this change intended for this pull request?

Reply:
Yes, this is part of ff2785a7 and related to https://github.com/hashicorp/terraform/issues/21070.

I think (but I'm not sure) that it is because Terraform 0.12 now expect the result of the apply to match the plan so updating the SDK highlighted a bug that was previously silent.